pymerc/game/recipe.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional, List
+from pymerc.api.models.common import Item, InventoryAccountAsset, InventoryManager
+from pymerc.api.models.static import Recipe as RecipeModel
+
+if TYPE_CHECKING:
+    from pymerc.client import Client
+    from pymerc.game.building import Building
+
+
+class Recipe:
+    def __init__(self, client: Client, recipe_name: str, recipe: RecipeModel | None = None):
+        self._client = client
+        self.recipe_name = recipe_name
+        self.labor: Optional[float] = 0.0
+        self.data: Optional[RecipeModel] = recipe
+        if recipe is not None:
+            self.data = recipe
+
+    async def load(self):
+        """Loads the data for the recipe and calculates labor required."""
+        if self.data is None:
```

Comment:
I would omit this. The idea is that someone can call `load()` repeatedly if they need to obtain up-to-date information.